- Primer entregable

Ernesto-1 opened the pull request:
- Se agregan modelos de datos
- Se agrega DataSource
- Se crea Web service
- Se crean repositorios por servicio
- Se crean ViewModel por servicio
- Se crean adapters para Home y listas en Detail
- Se agregan los fragments Home y Detail
- Se implementa viewModel en UI
- Se agrega estilo

Commit:
- Se estructura proyecto
- Se agregan modelos de datos
- Se agrega DataSource
- Se crea Web service
- Se crean repositorios por servicio
- Se crean ViewModel por servicio
- Se crean adapters para Home y listas en Detail
- Se agregan los fragments Home y Detail
- Se implementa viewModel en UI
- Se agrega estilo

ricsarabia commented on app/src/main/java/com/example/wizelineandroid/adapter/home/HomeAdapter.kt:
```diff
@@ -0,0 +1,62 @@
+            binding.txtn.text = item.book
+            binding.txtPminimo.text = item.minimum_price
+            binding.txtPmaximo.text = item.maximum_price
+            when(item.book){
```
You could get the icon from an API like https://cryptoicons.org/

ricsarabia commented on app/src/main/java/com/example/wizelineandroid/ui/home/HomeFragment.kt:
```diff
@@ -0,0 +1,66 @@
+                }
+                is Resource.Success -> {
+                    binding.shimmerViewContainer.stopShimmer()
+                    val filteredCoins = result.data.payload.filter {
```
Try to show all the available books

ricsarabia commented on app/src/main/java/com/example/wizelineandroid/utils/AppConstants.kt:
```diff
@@ -0,0 +1,8 @@
+object AppConstants {
+
+    //Constantes API
+    const val BASE_URL = "https://api.bitso.com/v3/"
```
URL constants would fit better in the WebService file

ricsarabia commented on app/src/main/java/com/example/wizelineandroid/adapter/home/HomeAdapter.kt:
```diff
@@ -0,0 +1,62 @@
+import com.example.wizelineandroid.data.model.ModelBook
+import com.example.wizelineandroid.databinding.ItemRowBinding
+
+class HomeAdapter(private val booksList: List<ModelBook>, private val itemClickListener: onUserClickListener): RecyclerView.Adapter<BaseViewHolder<*>>() {
```
Try to use the new List Adapter
https://ferdinandmarq.medium.com/why-you-should-be-using-the-new-and-improved-listadapter-in-android-17a2ab7ca644

Commit:
Actualizacion primer entregable
- Se agregan imagenes de forma dinamica
- Se eliminan LOGS
- se eliminan archivos png
